pkg/importer/util.go

```diff
@@ -42,3 +44,23 @@ func StreamDataToFile(dataReader io.Reader, filePath string) error {
 	}
 	return nil
 }
+
+// UnArchiveTar unarchives a tar file and streams its files
+// using the specified io.Reader to the specified destination.
+func UnArchiveTar(reader io.Reader, destDir string) error {
+	glog.V(1).Infof("begin untar...\n")
+	untar := exec.Command("/usr/bin/tar", "-xvC", destDir)
+	untar.Stdin = reader
+	var errBuf bytes.Buffer
+	untar.Stderr = &errBuf
+	err := untar.Start()
+	if err != nil {
+		return err
+	}
+	err = untar.Wait()
+	if err != nil {
+		glog.Errorf("%s\n", string(errBuf.Bytes()))
```

Comment:
Should probably have some sort of statement before dumping the errBuf here maybe?  Could probably include the err.Error() string for the error log, and a V3 or greater for the actual buffer.. but I might be wrong on that one.  I just think there at least needs to be something telling me what I'm looking at in the logs when you hit this error.

Reply:
Done, is that what you suggested?